README.md

```diff
@@ -19,7 +19,7 @@
 ### Installation
 
 * Installation
-    * Installing on [CoreOS / Linux distros](Documentation/deployment.md)
+    * Installing on [CoreOS Container Linux and other Linux distros](Documentation/deployment.md)
```

Comment:
nit: How about `[Container Linux / Other distros]`? In the rendered README, the link looks pretty long.